scripts/populate_mock_db.py

```
@@ -0,0 +1,63 @@
+                "skills": "TTR LGV BA",
+            }
+        )
+
+    insert_crew_details(crew_list, db_conn=conn)
+
+    now = datetime.now()
+    availability_data = []
+    for i in range(50):
+        slots = {}
+        # Make them available for today
+        for hour in range(0, 24):
```

Ruff (0.15.0)

[warning] Unnecessary `start` argument in `range`

Remove `start` argument

(PIE808)